common/djangoapps/edxmako/shortcuts.py

```diff
@@ -99,6 +99,17 @@ def microsite_footer_context_processor(request):
     )
 
 
+def custom_theme_context_processor(request):  # pylint: disable=unused-argument
+    """
+    Sets a boolean context variable to indicate whether the site uses a custom theme.
+    """
+    return dict(
+        [
+            ("THEME_ENABLED", settings.FEATURES.get("USE_CUSTOM_THEME", False))
+        ]
+    )
+
+
```

Comment:
Nit: how about

    return {"THEME_ENABLED": settings.FEATURES.get("USE_CUSTOM_THEME", False)}

or

    return dict(
        THEME_ENABLED=settings.FEATURES.get("USE_CUSTOM_THEME", False)
    )

just to reduce the number of parentheses a bit.